src/js/chartAuthoring.js

```diff
@@ -225,18 +324,20 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
     // Passing an empty dataset will clear and reset the form
     // Triggers change events to trigger model updating and propagation to other
     // interface elements
-    floe.chartAuthoring.updateDataEntryPanel = function (that, dataSet) {
+    floe.chartAuthoring.updateDataEntryPanelFromDataSet = function (that, dataSet) {
         // Clear any existing data entries
         var dataEntries = that.chartAuthoringInterface.dataEntryPanel.locate("dataEntry");
 
         var dataEntryLabelSelector = that.chartAuthoringInterface.dataEntryPanel.dataEntry.options.selectors.label;
         var dataEntryValueSelector = that.chartAuthoringInterface.dataEntryPanel.dataEntry.options.selectors.value;
 
-        dataEntries.each(function(idx) {
+        dataEntries.each(function (idx) {
             var currentData = dataSet[idx] !== undefined ? dataSet[idx] : {label: "", value: ""};
             $(this).find(dataEntryLabelSelector).val(currentData.label).trigger("change");
             $(this).find(dataEntryValueSelector).val(currentData.value).trigger("change");
         });
+
+        that.events.onDataAppliedToDataEntryPanel.fire();
```

Comment:
Noticed this event is only used in the test. It probably can be removed by refactoring the test to use ```func``` directly instead of listening to this event.